src/ids-base/ids-constants.js

```diff
@@ -44,14 +47,20 @@ export const props = {
   INDETERMINATE: 'indeterminate',
   KEEP_OPEN: 'keep-open',
   LABEL: 'label',
+  LABEL_AUDIBLE: 'label-audible',
   LABEL_REQUIRED: 'label-required',
   LABEL_FILETYPE: 'label-filetype',
+  MAX: 'max',
+  MAX_FILE_SIZE: 'max-file-size',
+  MAX_FILES: 'max-files',
+  MAX_FILES_IN_PROCESS: 'max-files-in-process',
   MAXLENGTH: 'maxlength',
   MENU: 'menu',
   MULTIPLE: 'multiple',
   NO_MARGINS: 'no-margins',
   NO_TEXT_ELLIPSIS: 'no-text-ellipsis',
   PLACEHOLDER: 'placeholder',
+  PARAM_NAME: 'param-name',
```

Comment:
```suggestion
  PARAM_NAME: 'param-name',
```
This name seems a little strange? Whats it for?

Reply:
This is the variable name to read from server